app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
```

Comment:
Kindly make sure that the views in devise for the registration and login have been modified to match the wireframes and their styling. You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.

Note: Even though styling is not required yet, that is different from applying attributes that will modify the layout of the elements to match the wireframe.

This is how the login page should look like:

![image](https://user-images.githubusercontent.com/38088818/233695046-36313f9e-4812-4532-a29f-35ac9ce48804.png)

- The input fields for email and password should be centered on the page, the login button should be the same width as the input fields for email and password.


Your login page:

![image](https://user-images.githubusercontent.com/38088818/233696399-abeb8993-cacd-40dd-a99a-71c82161e994.png)

- The same thing applies to the registration page as well...